src/chocolatey.resources/helpers/ChocolateyTabExpansion.ps1

```diff
@@ -135,6 +135,22 @@ function script:chocoCommands($filter) {
     $cmdList #| sort
 }
 
+function script:chocoApiKeysList() {
+    @(& $script:choco apikey list --limit-output) | ForEach-Object { $_.Split('|')[0] }

```

Comment:
During a meeting/chat we talked about including the headers to this call, in a similar manner that a different tab completion added them.

```suggestion
    @(& $script:choco apikey list --limit-output --include-headers)  |
            ConvertFrom-Csv -Delimiter '|'
```

THis is not the full example, you can either tack on `| Select-Object -ExpandProperty Id` to it, or update the caller to explicitly use the `Id` property.

THis is true for the other functions as well `chocoConfigsList`, `chocoFeaturesList`, `chocoLocalNonPinnedPackages`, `chocoLocalPinnedPackages`, `chocoRulesList`, `chocoSourcesList`, `chocoTemplatesList` as well as the existing `chocoRemotePackages`. (I can take care of the last one if you don't want to do it as part of this PR).

Reply:
I agree, this is a good idea!  I will get a new commit added to this PR to get this added.